Processes/Docs-Contribution-Process.md

```diff
@@ -0,0 +1,27 @@
+
+
+# Docs Contribution Process
+
+## Overview
+This document provides instructions for content contributors to the [EOS Documentation](https://docs.eosnetwork.com).
+
+ You can rely on the DevRel team to perform a thorough edit and review of your content before it is published to the EOS Documentation portal. The publication process includes GitHub workflows which publish Markdown (.md) formatted content.
+
+As a content contributor you are responsible for the accuracy and completeness of your submitted documentation. The DevRel team may contact you for clarification of your contribution, if necessary. You are also responsible to identify future changes for your content and let the DevRel team know of any information maintenance issues.
+
+As a content contributor you are also responsible to review the final submission before it is published. Your review is reflective of the accuracy and completeness of the content.
+
+
+### Stages of Publication
+1. Create branch
+2. Create and name document
+3. Apply template and write content
+4. Submit to DevRel
+5. Final review and approval
+
+Refer to the following procedures for guidance:
+* [How to write content for the EOS Documentation Portal](/processes/how-to-write-content-for-the-EOS-Documentation-Portal.md)
+
+* [How to submit content to the EOS Documentation Portal](processes/how-to-submit-content-to-the-EOS-documentation-portal.md)
```

Comment:
how-to-submit-content-to-the-EOS-documentation-portal.md -> how-to-submit-content-to-docs-portal.md

☝️ Make shorter filenames.

Reply:
Done